### PR TITLE
i18n: pages and messages

### PR DESCRIPTION
**Pull request by venjiang**
### i18n others

- delete account page 
- external page message
- mentions/feeds/settings page message 
- permalink page message 
- archive feed page message
- user config page message
- post page message

### CODE REVIEW

File: internal/langs/active.en.toml [hunk 1]
```diff
@@ -40,9 +40,15 @@ ConversationTitle = "Conversation"
...
 DefaultTwtPrompts = "What's on your mind?\nShare something insightful!\nGood day to you! What's new?\nDid something cool lately? Share it!\nHi! 👋 Don't forget to post a Twt today!"
+DeleteAccountFormConfirm = "Are you sure you want to delete your account and all feeds? This cannot be undone!"
+DeleteAccountFormDeleteAll = "Delete All"
+DeleteAccountNoFeedsSummary = "You do not have any feeds."
+DeleteAccountSummary = "Your account will be deleted permanently!"
+DeleteAccountTitle = "Delete Account"
 ErrorArchivingFeed = "Error archiving feed"
 ErrorCreateFeed = "Error creating: {{.Error}}"
 ErrorDeleteLastTwt = "Error deleting last twt"
+ErrorDeletingAccount = "An error occured whilst deleting your account"
```
Comment (der-lyse) [inline internal/langs/active.en.toml:51]: "occurred" is spelt with double "r".

**Review by der-lyse — CHANGES_REQUESTED** [review]
Apart from the copied typo this looks good to me. Disclaimer: I don't know anything about Chinese.

File: internal/langs/active.en.toml (hunk 1 above)
Comment (venjiang) [reply internal/langs/active.en.toml:51]: What happened?

File: internal/langs/active.en.toml [hunk 2]
```diff
@@ -40,7 +40,16 @@ ConversationTitle = "Conversation"
...
 DefaultTwtPrompts = "What's on your mind?\nShare something insightful!\nGood day to you! What's new?\nDid something cool lately? Share it!\nHi! 👋 Don't forget to post a Twt today!"
+DeleteAccountFormConfirm = "Are you sure you want to delete your account and all feeds? This cannot be undone!"
+DeleteAccountFormDeleteAll = "Delete All"
+DeleteAccountNoFeedsSummary = "You do not have any feeds."
+DeleteAccountSummary = "Your account will be deleted permanently!"
+DeleteAccountTitle = "Delete Account"
+ErrorArchivingFeed = "Error archiving feed"
 ErrorCreateFeed = "Error creating: {{.Error}}"
+ErrorDeleteLastTwt = "Error deleting last twt"
+ErrorDeletingAccount = "An error occured whilst deleting your account"
```
Comment (prologic) [inline internal/langs/active.en.toml:51]:
```suggestion
ErrorDeletingAccount = "An error occurred whilst deleting your account"
```

**Review by prologic — CHANGES_REQUESTED** [review]
Fix the typo or accept my suggested change and we're good to go :)

File: internal/langs/active.en.toml (hunk 2 above)
Comment (venjiang) [reply internal/langs/active.en.toml]: 🤝

Comment (venjiang) [reply internal/langs/active.en.toml]: The en message change causes the hash value of the message key in other languages to change. I will submit it again

**Review by prologic — APPROVED** [review]

**Comment by prologic** [comment]
cc @der-lyse

**Review by der-lyse — APPROVED** [review]
Commits look a bit messed up, but this doesn't matter, since it's all rebased and squashed anyways. Great work!

**Pull request merged**